### PR TITLE
addCampaigns

### PR DESCRIPTION
**Pull request by Nagham202**
Adding Campaigns

### CODE REVIEW

**Review by Nagham202 — COMMENTED** [review]
finallllly

File: app.ts
Comment (MoeShair) [inline app.ts:1]: why is line 3 deleted

File: app.ts [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-const port=3000;
-// Import routes
-app.use(express.json()); // Use built-in JSON parsing middleware
-const corsOptions = {
-  origin: 'http://localhost:4200',
-};
+const port = 3000;
 
-app.use(cors(corsOptions));
-app.use('/posts', postsRoutes);
```
Comment (MoeShair) [inline app.ts:15]: return this line

File: app.ts
Comment (Nagham202) [reply app.ts:1]: i don't know wallah ya partner

**Pull request merged**